conda/build_pytorch.sh

```diff
@@ -26,37 +26,67 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
     time conda build -c $ANACONDA_USER --no-anaconda-upload --python 2.7 pytorch-$BUILD_VERSION
     time conda build -c $ANACONDA_USER --no-anaconda-upload --python 3.5 pytorch-$BUILD_VERSION
     time conda build -c $ANACONDA_USER --no-anaconda-upload --python 3.6 pytorch-$BUILD_VERSION
+elif [[ "$OSTYPE" == "msys" ]]; then
+    conda uninstall -y conda-build
+    conda install -y conda-build=3.1.1 # The newest version will give "No module named torch" error
```

Comment:
It's not the conda-build's fault. Actually, it introduced a new method for writing meta.yaml. You can take a look at `C:\builder\conda\pytorch-cpu-0.3.1`. We should use newer version of conda-build.

Reply:
[Here](https://github.com/conda/conda-build/issues/2698#event-1488393700) is related issue I made.